spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -238,6 +238,7 @@
                                 due_date: Time.zone.today - 1.day,
                                 equipment_item: @model.equipment_items.first)
           r.save(validate: false)
+          @model.save
```

Comment:
Agreed... it looks like the counter cache callback is updating the local instance but not persisting it? Still weird.